crates/client/src/serialize/txt/parse_rdata.rs

```diff
@@ -69,7 +71,10 @@ impl RDataParser for RData {
                 return Err(ParseError::from("CDNSKEY should be dynamically generated"))
             }
             RecordType::KEY => return Err(ParseError::from("KEY should be dynamically generated")),
-            RecordType::DS => return Err(ParseError::from("DS should be dynamically generated")),
+            #[cfg(feature = "dnssec")]
+            RecordType::DS => RData::DNSSEC(DNSSECRData::DS(ds::parse(tokens)?)),
+            #[cfg(not(feature = "dnssec"))]
+            RecordType::CDS => return Err(ParseError::from("DS should be dynamically generated")),
```

Comment:
I think this cfg flag in the not case is meant to be DS, right?

Reply:
Right.
I have fixed that plus the other failures.